bin/hexdump

```diff
@@ -1,5 +1,19 @@
 #!/usr/bin/perl
 
+=begin metadata
+
+Name: hexdump
+Description: print input as hexadecimal
+Author: Michael Mikonos
+License: 
```

Comment:
This license can be artistic2 by default since it's a new program, but Michael can also choose one of the others from the LICENSE files in the distro.